### PR TITLE
Added a more modern styling for the website

### PR DESCRIPTION
**Pull request by darltrash**

### CODE REVIEW

File: website/style.css [hunk 1]
```diff
@@ -1,11 +1,13 @@
+@import url('https://fonts.googleapis.com/css2?family=PT+Sans&family=Source+Code+Pro&display=swap');
```
Comment (ikskuh) [inline website/style.css:1]: Can you please provide the used font files in the repo. It's not GDPR-compliant to use external google URLs

**Review by ikskuh — CHANGES_REQUESTED** [review]
Looks good overall, but the font thing is a problem

File: website/style.css (hunk 1 above)
Comment (darltrash) [reply website/style.css:1]: Ohhh, wait me a second, im downloading the stuff

**Comment by ikskuh** [comment]
Looks good! Let's merge!

**Pull request merged**